### PR TITLE
ISSUE-38 Queue support in subscription

### PR DESCRIPTION
**Pull request by josgilmo**

### CODE REVIEW

**Comment by dfeyer** [comment]
Any objection to merge this one ?

File: src/Connection.php [hunk 1]
```diff
@@ -287,13 +289,18 @@ public function publish($subject, $payload)
      *
      * @param string   $subject  Message topic.
      * @param \Closure $callback Closure to be executed as callback.
+     * @param string   $queue    Queue name for subscription.
      *
      * @return string
      */
-    public function subscribe($subject, \Closure $callback)
+    public function subscribe($subject, \Closure $callback, $queue = null)
```
Comment (dfeyer) [inline src/Connection.php]: Maybe implementation a new method queueSubscribe can help reading code, and allow the have a better method signature:

`public function queueSubscribe($subject, $queue, \Closure $callback)`

**Comment by repejota** [comment]
None from my part. 
Let me take a look and I will merge it 👍

**Comment by dfeyer** [comment]
@repejota I just push a implementation of the queueSubscribe method here https://github.com/repejota/phpnats/pull/45

**Pull request closed without merging**